packages/fiori/src/DynamicPageTitle.hbs

```diff
@@ -11,16 +11,19 @@
     {{/if}}
   </div>
 
-  <div class="{{classes.wrapper}}">
+  <div class="{{classes.wrapper}}"
+  @ui5-_minContentWidthChange={{onMinContentWidthChange}}>
```

Comment:
Minor: the name of the events seems to be defined in kebab-case, all over the project. Could we change this one also?